### DISCUSSION TITLE
Add Service Account Feature also! to bypass Limit's!

### DISCUSSION THREAD

**Discussion by arunpc4u** [dishapatel010/gdriveDL#2] [Ideas]
also Add Service Account Feature! to bypass Limit's!

**Comment by dishapatel010** [comment]
> also Add Service Account Feature! to bypass Limit's!

Will look into it if i decide to develop it fully. Right now you can enjoy getting [Transcoded URLs](https://github.com/dishapatel010/gdriveDL/commit/86b0b1a7d36c2d82ffcc7ff4fd05b8fe8e504c24)

**Reply by arunpc4u** [reply]
okay, thank you ♥:!

**Reply by OtakuFlix** [reply]
as an developer i built the limitless url if wanna know how then reach to me https://gdrivedl.aniflix-00.workers.dev/
